ember-app/app/vendor/jquery.color.contrast.js

```diff
@@ -4,5 +4,5 @@
     var r = this._rgba[0], g = this._rgba[1], b = this._rgba[2];
     return (((r*299)+(g*587)+(b*144))/1000) >= 131.5 ? "#333" : "white";
   };
-})(jQuery);
+})();
```

Comment:
Reverting the change and replacing `$` with `Ember.$` should satisfy jhint unless its not able to work with iffys